### PR TITLE
Guard for define in netcdf 4.9.0 and later.

### PR DESCRIPTION
**Pull request by danlipsa**
This code is already available in ParaView 5.11.0 so no patching needed there.

### CODE REVIEW

**Comment by danlipsa** [comment]
@kwryankrattiger @vicentebolea Please review

File: var/spack/repos/builtin/packages/paraview/package.py
Comment (vicentebolea) [inline var/spack/repos/builtin/packages/paraview/package.py]:
```suggestion
    patch("exodusII-netcdf4.9.0.patch", when="@:5.10.2")
```

This patch did not seem to make it to the 5.10.2 release.

**Review by vicentebolea — CHANGES_REQUESTED** [review]
proposed change in ver; looks good otherwise. Thanks for the contribution @danlipsa :+1:

File: var/spack/repos/builtin/packages/paraview/package.py
Comment (danlipsa) [reply var/spack/repos/builtin/packages/paraview/package.py]: True, I didn't see that version in spack so I did added it. I'll add it.

**Review by vicentebolea — APPROVED** [review]

**Comment by danlipsa** [comment]
@vicentebolea @kwryankrattiger Lets merge this, if you guys have merge permissions.

**Review by kwryankrattiger — APPROVED** [review]

**Pull request merged**
